packages/react/src/lib/applyAccessibilityKeyHandlers.ts

```diff
@@ -0,0 +1,18 @@
+import * as _ from 'lodash'
+
+import { Props } from '../types'
+import { KeyboardHandler, OnKeyDownHandler } from './accessibility/types'
+
+const applyAccessibilityKeyHandlers = (keyHandlers: OnKeyDownHandler, passedProps: Props) =>
```

Comment:
Is the second argument really always going to be `Props`? If we want to use this functionality with the slots as well, should we receive here `ShorthandValue`? I know that the Popup has special handling of the `triggerProps`, but usually this is not the case...

Reply:
Done, it makes sense because we also deal with slots 👍